paper-input-container.html

```diff
@@ -177,9 +176,10 @@
       }
 
       .input-content {
-        position: relative;
         @apply(--layout-horizontal);
         @apply(--layout-center);
+
+        position: relative;
```

Comment:
Yeah a style guide would be neat. I think this particular change is ok, but moving CSS around just for the sake of moving it makes me a bit worried, since sometimes before or after a mixin has a meaning (`--layout-fit`, for example, applies `position:relative`, so in some cases if you want to override that, before or after the mixin is applied is actually important)